src/actions/index.js

```diff
@@ -6,3 +6,24 @@
 // our action creator will be a function that returns a function
 // the url to fetch characters from is `https://swapi.co/api/people/`
 // remember that now we have controll over our thunk-based action creator
+
+import Axios from 'axios';
+
+export const FETCHING = 'FETCHING';
+export const SUCCESS = 'SUCCESS';
+export const ERROR = 'ERROR';
+
+
+export function getStarwarsCharacters() {
+  return dispatch => {
+    dispatch({ type: FETCHING });
+    
+    Axios.get(`https://swapi.co/api/people/`)
+    .then(response => {
+      dispatch({ type: SUCCESS, starWarsData: response.data.results });
+    })
+    .catch(error => {
+      dispatch({ type: ERROR, errorMessage: "Oops Something Went Wrong. Please Try Again!" });
```

Comment:
While I like that you put your own error message in here, the argument passed (`error`) to your .catch callback function, might have useful information that you could be preventing from showing here. Having your own error message is good, but try to figure out a way to have the error that gets passed through displayed as well.